wire-ios-sync-engine/Source/Synchronization/Strategies/TeamInvitationRequestStrategy.swift

```diff
@@ -77,7 +77,7 @@ public final class TeamInvitationRequestStrategy: AbstractRequestStrategy {
         )
 
         request.add(ZMCompletionHandler(on: managedObjectContext, block: { [weak self] response in
-            self?.processResponse(response, for: email)
+            self?.processResponse(response, for: email) // TODO: handle new error? 403 "condition-failed"
```

Comment:
Is there something to handle or would it end up as an error log?

Reply:
Agreed, I don't think we have tickets or design changes to handle this error

Reply:
removed